api/specs/api-v2.0.yaml

```
@@ -0,0 +1,1078 @@
+  version: '2.0'
+  termsOfService: https://doc.covoiturage.beta.gouv.fr/le-registre-de-preuve-de-covoiturage/kit-de-communication/cgu-conditions-generales-dutilisation
+  contact:
+    name: Équipe technique
+    email: technique@covoiturage.beta.gouv.fr
+  license:
+    name: Apache 2
+    url: https://raw.githubusercontent.com/betagouv/preuve-covoiturage/main/LICENSE
+
+externalDocs:
+  url: https://doc.covoiturage.beta.gouv.fr/
+  description: Documentation générale 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)